examples/account/main.go

```diff
@@ -35,10 +35,10 @@ func main() {
 	fmt.Println("Counter is currently at: ", callResp[0])
 
 	// init account handler
-	ks := caigo.NewMemKeystore()
+	ks := starknet.go.NewMemKeystore()
```

Comment:
Wouldn't this cause conflict? Also package name in previous files is `starknetgo`
This repeats other times as well

Reply:
Yes this is an error. I thought I caught all of these! Thanks, I'll have another look now

Reply:
I fixed the import errors in examples, but it's worth noting that they are still broken and should be fixed in following PRs